storage/dvo_recommendations_storage.go

```diff
@@ -723,3 +724,51 @@ func (storage DVORecommendationsDBStorage) WriteHeartbeat(
 
 	return err
 }
+
+// WriteHeartbeats insert multiple heartbeats
+func (storage DVORecommendationsDBStorage) WriteHeartbeats(
```

Comment:
wouldn't it make sense to remove `WriteHeartbeat` and replace with `WriteHeartbeats`? in the end they do the same

Reply:
Well, my plan was to convert it later to UpdateHeartbeat, but OK. I will remove it.

Reply:
Done